examples/example_plugin/example_plugin/ui/index.js

```diff
@@ -1,11 +1,32 @@
 export { default as ExampleAppFullWidthPageComponent } from "./FullWidthPage"
+export { default as ExamplePluginNonModelView } from "./NonModelView"
 
 const app_config = {
     detail_tabs: {},
     full_width_components: {
         "dcim:sites": ["ExampleAppFullWidthPageComponent"]
     },
-    view_overrides: {}
+    view_overrides: {},
+    routes: [
+        {
+            name: "Inventory",
+            groups: [
+                {
+                    name: "Example App",
+                    weight: 150,
+                    items: [
+                        {
+                            name: "Non Model View",
+                            weight: 100,
+                            path: "/non-model-view/",
+                            component: "ExamplePluginNonModelView",
+                            namespace: "non-model-view", 
```

Comment:
Please add some explanatory comments where appropriate, at least:

- is the `path` absolute, or relative to the `plugins/example-plugin/` base path? (Do we support both?)
- why is `component` a string instead of an object reference?
- what purpose does `namespace` serve?